flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -154,6 +203,11 @@ class FlxGamepad implements IFlxDestroyable
 		return gamepadButton;
 	}
 	
+	public inline function getFlipAxis(AxisID:Int):Int
```

Comment:
Okay maybe I'm crazy, but on my computer chrome doesn't flip the Y axis in flash (so manually inverting it messes it up)... only on the standalone windows player do I have to flip it. 

It even says manufacturer = "Google Pepper" in the log trace in flash/chrome, so I dunno...

I'll commit what works here and you tell me what's happening for you.